R/addLDA.R

```diff
@@ -0,0 +1,113 @@
+#' Latent Dirichlet Allocation
+#'
+#' These functions perform Latent Dirichlet Allocation on data stored in a 
+#'  \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'  object.
+#' 
+#' @param x a \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'  object.
+#'   
+#' @param k Integer scalar. A number of latent vectors/topics. 
```

Comment:
Looks good. Can you still put the class inside  \code{] to make it nicer. --> \code{Integer scalar} (Check other also)